### ISSUE TITLE
GPG key expired - Installation from repo not possible

### ISSUE THREAD

**Issue by Kimbaras** [OCSInventory-NG/OCSInventory-Server#432]
*It's hard to solve a problem when important details are missing, that why we added this template, to help you and us.*

### General informations
Operating system : Ubuntu / Debian

### Server informations
Perl version : Not relevant
Mysql / Mariadb / Percona version :  Not relevant

### OCS Inventory informations
Ocs server version : 2.10 (the latest one release on the dedicated repository on deb.ocsinventory-ng.org)

### Problem's description
*Describe your problem here*

First of all, not sure if it's the right place to report this, if not sorry, and please provide me the right place to report it.

Simply put, the gpg key is expired so the installation from the repository (as detailed at the docs: https://wiki.ocsinventory-ng.org/03.Basic-documentation/Setting-up-the-UNIX-agent-using-repository-on-client-computers/ - Installing UNIX Agent with APT step) is not possible.

<img width="548" alt="Schermata 2023-08-09 alle 15 25 01" src="https://github.com/OCSInventory-NG/OCSInventory-Server/assets/102742484/698967cf-6fee-4e81-b2ee-aa3d36796bac">

Any ETA for the new key?

Thanks!

**Comment by Lea9250** [comment]
Hi @Kimbaras ,

Thank you for reporting it, the key was renewed. I'll close this issue now.

Best regards,
Léa
